packages/fxa-settings/src/components/PageSettings/index.stories.tsx

```diff
@@ -3,70 +3,89 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from 'react';
-import { storiesOf } from '@storybook/react';
+import { Account } from '../../models/Account';
+
 import { PageSettings } from '.';
+import { Config } from '../../lib/config';
+
 import { LocationProvider } from '@reach/router';
 import AppLayout from '../AppLayout';
 import { isMobileDevice } from '../../lib/utilities';
 import { mockAppContext, mockEmail, MOCK_ACCOUNT } from '../../models/mocks';
 import { MOCK_SERVICES } from '../ConnectedServices/mocks';
 import { AppContext } from 'fxa-settings/src/models';
+import { MOCK_LINKED_ACCOUNTS } from '../LinkedAccounts/mocks';
+import { Meta } from '@storybook/react';
+
 
 const SERVICES_NON_MOBILE = MOCK_SERVICES.filter((d) => !isMobileDevice(d));
 
-storiesOf('Pages/Settings', module)
-  .addDecorator((getStory) => <LocationProvider>{getStory()}</LocationProvider>)
-  .add('cold start', () => (
-    <AppContext.Provider
-      value={mockAppContext({
-        account: {
-          ...MOCK_ACCOUNT,
-          displayName: null,
-          avatar: { id: null, url: null },
-          recoveryKey: false,
-          totp: { exists: false, verified: false },
-          attachedClients: SERVICES_NON_MOBILE,
-        } as any,
-      })}
-    >
-      <AppLayout>
-        <PageSettings />
-      </AppLayout>
-    </AppContext.Provider>
-  ))
-  .add('partially filled out', () => (
-    <AppContext.Provider
-      value={mockAppContext({
-        account: {
-          ...MOCK_ACCOUNT,
-          displayName: null,
-          totp: { exists: true, verified: false },
-          attachedClients: SERVICES_NON_MOBILE,
-        } as any,
-      })}
-    >
-      <AppLayout>
-        <PageSettings />
-      </AppLayout>
-    </AppContext.Provider>
-  ))
+export default {
+  title: 'Pages/Settings',
+  component: PageSettings,
+} as Meta;
+
+const coldStartAccount = {
+  ...MOCK_ACCOUNT,
+  displayName: null,
+  avatar: { id: null, url: null },
+  recoveryKey: false,
+  totp: { exists: false, verified: false },
+  attachedClients: [SERVICES_NON_MOBILE[0]],
+} as unknown as Account;
+
+const partiallyFilledOutAccount = {
+  ...MOCK_ACCOUNT,
+  displayName: null,
+  totp: { exists: true, verified: false },
+  attachedClients: SERVICES_NON_MOBILE,
+  linkedAccounts: MOCK_LINKED_ACCOUNTS,
+} as unknown as Account;
+
+const completelyFilledOutAccount = {
+  ...MOCK_ACCOUNT,
+  subscriptions: [{ created: 1, productName: 'x' }],
+  emails: [
+    mockEmail('johndope@example.com'),
+    mockEmail('johndope2@gmail.com', false),
+  ],
+  attachedClients: SERVICES_NON_MOBILE,
+  linkedAccounts: MOCK_LINKED_ACCOUNTS,
+}
+
+const storyWithContext = (
```

Comment:
Just thinking outloud; if we end up using `storyWithContext` in our other SB refactors, we should definitely consider pulling this out into a helper function to use across stories, we can have it take a `children` prop.

Reply:
Mm I like that idea. It would feel nice to do a big sweep refactoring all of these and refactor it out then